recipe/meta.yaml

```diff
@@ -205,6 +216,12 @@ outputs:
         - test -f ${PREFIX}/lib/libglib-2.0${SHLIB_EXT}             # [unix]
         - glib-compile-resources --help
         - gobject-query --help
+        - head ${PREFIX}/bin/glib-mkenums               # [unix]
+        - head ${PREFIX}/bin/glib-genmarshal            # [unix]
+        - head ${PREFIX}/bin/gtester-report             # [unix]
+        - head -n 1 ${PREFIX}/bin/glib-mkenums | grep -E "(${PREFIX}|/usr/bin/env python)"      # [unix]
+        - head -n 1 ${PREFIX}/bin/glib-genmarshal | grep -E "(${PREFIX}|/usr/bin/env python)"   # [unix]
+        - head -n 1 ${PREFIX}/bin/gtester-report | grep -E "(${PREFIX}|/usr/bin/env python)"    # [unix]
```

Comment:
So i don't really understand why or how this became /usr/bin/env instead of the ${PREFIX}...

Reply:
maybe it is a detail with how noarch python works?

Reply:
Might it not be because it's supposed to point to the environment Python when installed instead of some strange prefix only used during build?